### PR TITLE
[Docs]Update dev.md to Chinese

### PR DESCRIPTION
**Pull request by infdahai**
Signed-off-by: cluster <clusternlh@gmail.com>

### CODE REVIEW

File: docs/book/zh/src/dev.md [hunk 1]
```diff
@@ -1 +1,11 @@
 # 开发 WasmEdge 应用
+
+WebAssembly 技术关键价值在于它支持多种编程语言。
+WebAssembly是一种为多种语言服务的托管运行时，包括 C/C++, Rust, Go,Swift, Kotlin, AssemblyScript, Grain and even JavaScript and Python.
```
Comment (hydai) [inline docs/book/zh/src/dev.md:4]: 英文跟中文字之間應該有一個半形空格，如 `WebAssembly 是`.

File: docs/book/zh/src/dev.md [hunk 2]
```diff
@@ -1 +1,11 @@
...
+
+* 对于编译型语言(例如C和Rust), WasmEdge WebAssembly 提供了一个安全、可靠、隔离和容器化的运行时，
```
Comment (hydai) [inline docs/book/zh/src/dev.md:6]: 使用中文時，應用全形符號，如 `，` 而非 `,`

File: docs/book/zh/src/dev.md [hunk 3]
```diff
@@ -1 +1,11 @@
...
+而非Native Client.
```
Comment (hydai) [inline docs/book/zh/src/dev.md:7]: 同理 `。` 而非 `.`

**Review by hydai — CHANGES_REQUESTED** [review]

File: docs/book/zh/src/dev.md (hunk 3 above)
Comment (infdahai) [reply docs/book/zh/src/dev.md:7]: 好的，晚点我会修改并提交更多的内容。

File: docs/book/zh/src/dev/go.md [hunk 4]
```diff
@@ -1 +1,119 @@
 # Go
+
+在 WasmEdge 中运行 Go 程序最好的方式是使用 [TinyGo](https://tinygo.org/) 编译 Go 源码到 WebAssembly。 在本篇中，我们将向你介绍这种做法。
+
+## 安装 TinyGo
+
+你必须确保在安装 TinyGo 之前已经阅读 [Go already installed](https://go.dev/doc/install) 这篇文章并安装 Go 到你的机器上。
+
```
Comment (alabulei1) [inline docs/book/zh/src/dev/go.md:8]: 这里应该只是说，在安装 TinyGo 之前，请确保已经在机器上安装了 Go。 Go already installed  的 link 只是说明 go 的安装文档

File: docs/book/zh/src/dev/go.md [hunk 5]
```diff
@@ -1 +1,119 @@
...
+### 编译和构建
+
+之后，使用 TinyGO 编译 `main.go` 程序到 WebAssembly 。
+
+```bash
+tinygo build -o hello.wasm -target wasi main.go
+```
+
+你将在同一目录下一个名为 `hello.wasm` 文件。这是一个 WebAssembly 字节码文件。
```
Comment (alabulei1) [inline docs/book/zh/src/dev/go.md:46]: 你将在同一目录看见，少了 see。

File: docs/book/zh/src/dev/js.md [hunk 6]
```diff
@@ -1 +1,44 @@
...
+
+在这一章节，我们将演示如何在 WasmEdge 中运行 JavaScript 并增强(它的性能)。
+
+* [起步](js/quickstart.md) 描述了怎样在 WasmEdge 中运行简单的 JavaScript 程序。
+* [ES6 模块](js/es6.md) 展示了怎样在 WasmEdge 中运行 ES6 模块。
+* [CommonJS 模块](js/cjs.md) 展示了怎样在 WasmEdge 中运行 CommonJS 模块。
+* [NodeJS 和 NPM 模块](js/npm.md) 展示了怎样在 WasmEdge 中运行 NPM 模块。 
+* [React SSR](js/ssr.md) 展示了在 WasmEdge 中一个 React SSR 应用实例。
+* [TensorFlow](js/tensorflow.md) 展示了怎样通过 JavaScript API 去使用 WasmEdge 的 Tensorflow 扩展。
+* [网络套接字](js/networking.md) 展示了使用 WasmEdge 网络扩展和提供的 JavaScript API 去构建 HTTP 客户端和服务端程序。
```
Comment (alabulei1) [inline docs/book/zh/src/dev/js.md:23]: 1. Socket 应该不翻译，网络 socket
2. 展示了如何使用 WasmEdge 网络扩展和及其提供的 JavaScript API

File: docs/book/zh/src/dev/python.md [hunk 7]
```diff
@@ -1 +1,48 @@
...
+然后你可以使用下面的命令去下载并编译 RustPython。
+
+```
+$ git clone https://github.com/RustPython/RustPython.git
+$ cd RustPython
+$ cargo build --release --target wasm32-wasi --features="freeze-stdlib"
+```
+
+启用 `freeze-stdlib` 功能可以使 Python 标准库塞进二进制文件中。输出文件路径为 `target/wasm32-wasi/release/rustpython.wasm`。
+
```
Comment (alabulei1) [inline docs/book/zh/src/dev/python.md:22]: 启用 `freeze-stdlib` 功能可以在二进制文件中使用 Python 标准库。 这样更通顺一些?

**Comment by alabulei1** [comment]
Hi @infdahai 
Thanks for your contribution. Much appreciated. 
You did a lot of work. I need some time to take a look at your code. Will come back soon.

File: docs/book/zh/src/dev/go.md (hunk 4 above)
Comment (infdahai) [reply docs/book/zh/src/dev/go.md:8]: 这里我更改成
```md
你需要在安装 TinyGo 之前已经在你的机器上安装过 [Go](https://go.dev/doc/install)
```

File: docs/book/zh/src/dev/go.md [hunk 8]
```diff
@@ -1 +1,119 @@
...
+```go
+package main
+
+func main() {
+    println("Hello TinyGo from WasmEdge!")
+}
+``` 
+
+> 在 `main()` 函数中，你可以使用 GO 标准 API 读写文件，
+> 并访问命令行参数和 `env` 变量。
```
[the remark this answers is not in the document]
Comment (infdahai) [reply docs/book/zh/src/dev/go.md:36]: 已修改到同一行上。

File: docs/book/zh/src/dev/js.md (hunk 6 above)
Comment (infdahai) [reply docs/book/zh/src/dev/js.md:23]: 已修改

File: docs/book/zh/src/dev/go.md (hunk 5 above)
Comment (infdahai) [reply docs/book/zh/src/dev/go.md:46]: 补上了。

File: docs/book/zh/src/dev/python.md (hunk 7 above)
Comment (infdahai) [reply docs/book/zh/src/dev/python.md:22]: 已修改，这里口胡了。

**Comment by infdahai** [comment]
Ok, I will fix some errors basis on reviews.

**Comment by alabulei1** [comment]
@infdahai  Thanks for your contribution too. I finished my review. Just need to make some mirror adjustments.

**Comment by alabulei1** [comment]
@infdahai  Happy New Year! Do you have time to revise those feedback recently?

**Comment by infdahai** [comment]
@alabulei1 I fix conflicts and hope you check again.

File: docs/book/zh/src/dev/rust.md [hunk 9]
```diff
@@ -1 +1,112 @@
 # Rust
+
+Rust 是 WebAssembly 生态系统中位居“一等公民”的编程语言之一。 WebAssembly 全部的 WasmEdge 扩展都为开发者提供 Rust APIs。在本篇中，我们将展示如何编译你的 Rust 应用到 wasm 字节码并在 WasmEdge 运行时上运行。
```
[the remark this answers is not in the document]
Comment (alabulei1) [reply docs/book/zh/src/dev/rust.md:3]: @infdahai  从这里开始的 comment 没有调整哈，请看一下，谢谢。

**Comment by infdahai** [comment]
Ok, I will fix these problems.

**Comment by alabulei1** [comment]
@infdahai We really want to merge your PR first. Could you solve my comments and fix the conflicts?

**Comment by infdahai** [comment]
OK, I will fix these.

**Comment by infdahai** [comment]
Please check again. I fix some conficts based on those previous changes.

File: docs/book/zh/src/dev/js.md [hunk 10]
```diff
@@ -1 +1,44 @@
...
+
+## 关于 v8 的说明
+
+现在选择 QuickJS 作为我们的 JavaScript 引擎可能引发性能的问题。由于缺乏 JIT 支持， QuickJS 是否比 v8 [慢很多](https://bellard.org/quickjs/bench.html) ？是的，但是我想说说选择 QuickJS 的理由。
+
+第一点，QuickJS 比 v8 资源消耗小很多。事实上，它仅仅占用 v8 所消耗资源的 1/40（或 2.5%)。你可以在单机上运行比 v8 函数更多的 QuickJS 函数。
+
+第二点，对于大多数业务逻辑应用，原生性能并不关键。应用程序可能具有计算密集型任务，比如 AI 动态推理。 WasmEdge 允许 QuickJS 应用使用高性能 WebAssembly 来完成这些任务， 而使用 v8 添加这些扩展模块并不容易。
+
+第三点， WasmEdge 是 [一种 OCI 兼容的容器](../kubernetes.md)。它默认安全，支持资源隔离，并且可以通过容器管理工具在单个 k8s 集群中与 Linux 容器上并行。
```
Comment (alabulei1) [inline docs/book/zh/src/dev/js.md:35]: 是 [一种 OCI 兼容的容器] 不用空格哈

File: docs/book/zh/src/dev/js.md [hunk 11]
```diff
@@ -1 +1,44 @@
...
+
+最后， v8 攻击面广泛，因此需要付出 [大量努力](https://blog.cloudflare.com/mitigating-spectre-and-other-security-threats-the-cloudflare-workers-security-model/) 才能使其安全运行在公共云环境上。众所周知，[许多 JavaScript 安全问题由 JIT 引起](https://www.theregister.com/2021/08/06/edge_super_duper_security_mode/) 。所以，也许在云原生环境下关闭 JIT 未必是坏事！
```
Comment (alabulei1) [inline docs/book/zh/src/dev/js.md:37]: 付出 [大量努力] 中文前后不用空格

公共云改成公有云吧

**Review by alabulei1 — COMMENTED** [review]
Hi, @infdahai  Thanks for your work. I have left some comments, please check out.

Also, I'm wondering if you need to delete the dev/rust, dev/go, and dev/python, because #1483, #1482, and #1402 have been merged. Let me know what do you think. Thanks.

**Pull request closed without merging**